test/ema.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-env mocha */
+'use strict'
+
+const assert = require('assert')
+const EMA = require('../lib/ema')
+
+const emaPeriod = [10]
+const candles = [{
```

Comment:
interesting, we have a seed period of 10, but just feed two candles.

how about this scenario, where we add ema values but only the last 2 should be counted in:

```js
const emaPeriod = [2]
candles.length // 3 
```

Reply:
see also other question... should we return `null` in those cases?

Reply:
done accordingly